README.md

```diff
@@ -211,9 +211,14 @@ The `express-cli` also comes with additional utility commands, listed below. Som
   - Rewinds the chain by a defined number of blocks (not greater than `128`). Default `numberOfBlocks` value is `100`.
 
 - `../../bin/express-cli --eip-1559-test [index]`
+
   - Executes a test to send EIP 1559 tx. In case of a non-dockerized devnet, if an integer [index] is specified, it will use
     that VM to send the tx. Otherwise, it will target the first VM.
 
+- `../../bin/express-cli --shadow-fork [block]`
```

Comment:
Can we call the option param `blockNumber` ?

Reply:
Made the change: [88a5694](https://github.com/maticnetwork/matic-cli/pull/157/commits/88a56943bc7a38b12a738340a89a6db70e4bd69f)